src/main/java/com/google/cloud/genomics/dataflow/pipelines/ShardedBAMWriting.java

```diff
@@ -225,9 +208,4 @@ public Contig apply(String str) throws Exception {
           return Contig.parseContigsFromCommandLine(str).iterator().next();
```

Comment:
s/str/contigStr

It will require the following update to this line from:

```Java
public Contig apply(String str) throws Exception {
```
to
```Java
public Contig apply(String contigStr) throws Exception {
```

This is to keep it consistent with [WriteReadsTransform](https://github.com/googlegenomics/dataflow-java/blob/de9258adc9bd8523d35058f43446b52e60332a27/src/main/java/com/google/cloud/genomics/dataflow/writers/WriteReadsTransform.java#L118)